index.html

```diff
@@ -9,11 +9,66 @@
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
-<body>
+
 
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
+<body>
+    <header>
+       
+    <!--Navigation--> 
+    <nav>    
+        <a href="#" class="logo"> 
+           <img src="img/karma-logo.svg" alt="logo">
+        </a>     
+        <a href="">Meet Karma</a>
+        <a href="">How it works</a>
+        <a href="">Store</a>
+        <a href="">Blog</a>
+        <a href="">Help</a>
+        <a href="">Log in</a>
+    </nav>
+
+    <!--Hero-->
+    <section class="hero-image"></section>
+        <div class="hero-text">
```

Comment:
```suggestion
    <section class="hero-text">
```
With the flexbox suggestions I gave for the hero section, we don't need both a section and div so we can delete the div